packages/ui/src/styles/grid/_grid-variables.scss

```
@@ -1,27 +1,44 @@
-// Breakpoints are in px because they're bound to screen size, which is in px.
-// Everything else is in rem.
+// variables for runtime calculations, --media-ratio variable is overwritten by breakpoints service
+:root {
+  --va-media-ratio-default: var(--va-media-ratio, 1);
+  --va-grid-gutter-default: 0.5rem;
+  --va-grid-gutter-base: calc(var(--va-media-ratio-default) * var(--va-grid-gutter-default));
+  --va-grid-spacing-default: 1rem;
+  --va-grid-spacing-base: calc(var(--va-media-ratio-default) * var(--va-grid-spacing-default));
+}
+
+// breakpoints must be always synchronized with breakpoints service defaults
 $grid-breakpoints: (
   xs: 0,
-  sm: 576px,
-  md: 768px,
-  lg: 992px,
-  xl: 1200px,
+  sm: 600px,
+  md: 1024px,
+  lg: 1440px,
+  xl: 1920px,
 ) !default;
 
-$grid-gutter-default: 0.5rem !default;
 $grid-columns: 12 !default;
 
-// Gutters correspond to $grid-breakpoints
+// gutters are depending on the current breakpoint (provided by breakpoints service via --media-ratio)
 $grid-gutters: (
-  xs: $grid-gutter-default * 0.25,
-  sm: $grid-gutter-default * 0.5,
-  md: $grid-gutter-default,
-  lg: $grid-gutter-default * 2,
-  xl: $grid-gutter-default * 3,
+  1: #{calc(var(--va-grid-gutter-base) * 0.25)},
+  2: #{calc(var(--va-grid-gutter-base) * 0.5)},
+  3: #{var(--va-grid-gutter-base)},
+  4: #{calc(var(--va-grid-gutter-base) * 2)},
+  5: #{calc(var(--va-grid-gutter-base) * 3)},
 ) !default;
 
+// spacers are used for various paddings and typings to make them consistent, also depending on current breakpoint
+$spacers: (
+  0: 0,
+  1: #{calc(var(--va-grid-spacing-base) * 0.25)},
+  2: #{calc(var(--va-grid-spacing-base) * 0.5)},
+  3: #{var(--va-grid-spacing-base)},
+  4: #{calc(var(--va-grid-spacing-base) * 1.5)},
```

```suggestion
  4: #{calc(var(--va-grid-spacing-base) * 2)},
```
should be in line with the grid-gutter